driver-core/src/main/com/mongodb/client/model/expressions/DocumentExpression.java

```diff
@@ -16,6 +16,11 @@
 
 package com.mongodb.client.model.expressions;
 
+/**
+ * Expresses a document value. A document is an ordered set of fields, which may
+ * be of an arbitrary string value, that each map to a value of any other
```

Comment:
> fields, which may be of an arbitrary string value

Field values are not necessary strings.

Reply:
I mean that the fields (names) are an arbitrary string value (though I am not sure about this, for example how is whitespace handled).